velox/functions/prestosql/window/tests/LeadLagTest.cpp

```diff
@@ -104,6 +110,29 @@ TEST_P(LeadLagTest, offset) {
   assertQuery(queryInfo.planNode, expected);
 }
 
+TEST_P(LeadLagTest, ignoreNullsInt64Offset) {
+  // The offset is bigger than int32:max() and it is also a positive number
+  // if cast to int32. With only such a special number we can trigger
+  // some tricky bug.
+  int64_t offset = (int64_t)std::numeric_limits<uint32_t>::max() + 2;
+  auto data = makeRowVector(
+      {// Values.
+       makeNullableFlatVector<int64_t>({1, std::nullopt, 3, 4, 5}),
+       // Offsets.
+       makeFlatVector<int64_t>({offset, offset, offset, offset, offset})});
+
+  createDuckDbTable({data});
+
+  auto assertResults = [&](const std::string& functionSql) {
```

Comment:
This lambda is used only once, hence, not needed. Let's remove for readability.

Reply:
Added another CONSTANT test, so will keep it as a lambda.